CMakeLists.txt

```diff
@@ -1016,7 +1016,8 @@ else()
     port/port_posix.cc
     env/env_posix.cc
     env/fs_posix.cc
-    env/io_posix.cc)
+    env/io_posix.cc
+          env/flink/JavaUtil.cc)
```

Comment:
How about naming java_util.cc which is consistent with other files ?

Reply:
This class has been renamed to jvm_util.cc